### PR TITLE
Implemented Client side validation

### PR DESCRIPTION
**Pull request by Asim-Khaleel**
PR for [Client-side Validation](https://dev.azure.com/CodeNinjaConsulting/Talent%20Pool/_workitems/edit/1654/)

### CODE REVIEW

File: Task1/Portfolio.js
Comment (aaminbaig) [inline Task1/Portfolio.js:1]: Why does this PR have your Task1 files?

File: Task2/Task2.html
Comment (aaminbaig) [inline Task2/Task2.html:1]: Why are you naming files on the bases of your task? Properly name them.

File: Task1/Task1.html [hunk 1]
```diff
@@ -8,7 +8,9 @@
 
   <body>
     <div class="informationContainer">
-      <div class="loginSection"><span class="loginText">Login</span></div>
+      <div class="loginSection">
+        <button class="loginText" id="loginButton">Login</button>
```
Comment (aaminbaig) [inline Task1/Task1.html]: handle the button with onClick

File: Task1/Portfolio.js
Comment (aaminbaig) [reply Task1/Portfolio.js:1]: Please update the naming convention. Its confusing me.

**Review by aaminbaig — CHANGES_REQUESTED** [review]

File: Task1/Portfolio.js
Comment (Asim-Khaleel) [reply Task1/Portfolio.js:1]: 1) PR have the task 1 file because I had to click the button on task 1 to get here to the signup page

**Review by aaminbaig — APPROVED** [review]

**Pull request merged**